.wp-env.json

```diff
@@ -3,6 +3,9 @@
 	"plugins": [ "." ],
 	"env": {
 		"tests": {
+      "config": {
+        "FS_METHOD": "direct"
+      },
```

Comment:
Need to fix indentation.

Reply:
It's actually the other indentation that was incorrect, not this one :)

Fixed it.